README.md

```diff
@@ -18,121 +14,101 @@ an exact solution if possible or use an appropriate numeric solver otherwise.
-| -module_name  | (Optional) Sets the name of the module which shall be generated. Default is the name of the directory containing the models. |
+| -module_name  | (Optional) Sets the name of the module which shall be generated. Default is the name of the directory containing the models. The name has to end in "module". Default is `nestmlmodule`. |
 | -store_log    | (Optional) Stores a log.txt containing all messages in JSON notation. Default is OFF.|
 | -dev          | (Optional) Executes the toolchain in the development mode where errors in models are ignored. Default is OFF.|
 
-
-Generated artifacts are copied to the selected target directory (default is /target). In order to install 
-the models into NEST, the following commands have to be executed:
+Generated artifacts are copied to the selected target directory (default is `target`). In order to install the models into NEST, the following commands have to be executed from within the target directory:
 ```
 cmake -Dwith-nest=<nest_install_dir>/bin/nest-config .
 make all
 make install
 ```
-where _nest\_install\_dir_ points to the installation directory of NEST (e.g. work/nest-install). Subsequently, PyNEST can be used to set up and execute a simulation.
-
-PyNestML is also available as a component and can therefore be used from within 
-other Python tools and scripts. After the PyNestML has been installed,
-the following modules have to be imported:
-```
-from pynestml.frontend.pynestml_frontend import to_nest,install_nest
+where `<nest_install_dir>` is the installation directory of NEST (e.g. `/home/nest/work/nest-install`). Subsequently, the module can either be linked into NEST (see [Writing an extension module](https://nest.github.io/nest-simulator/extension_modules)), or loaded dynamically using the `Install` API function. For example, to dynamic load a module with `module_name` = `mymodelsmodule` in PyNEST:
+```py
+nest.Install("mymodelsmodule")
 ```
-Subsequently, it is possible to call PyNestML from other Python tools and scripts via:
 
+PyNestML is also available as a component and can therefore be used from within other Python tools and scripts. After PyNestML has been installed, the following modules have to be imported:
+```py
+from pynestml.frontend.pynestml_frontend import to_nest, install_nest
 ```
+Subsequently, it is possible to call PyNestML from other Python tools and scripts via:
+```py
 to_nest(path, target, dry, logging_level, module_name, store_log, dev)    
 ```
-This operation expects the same set of arguments as in the case of the shell/CMD call,
-with the following default values being used, where only the __path__ is mandatory:
+This operation expects the same set of arguments as in the case of command line invocation. The following default values are used corresponding to the command-line defaults; note that only `path` is mandatory:
```

Comment:
Make spelling of `command line` consistent. There's on with and one without hyphen in this sentence.